bitnami/moodle/README.md

```diff
@@ -52,140 +52,141 @@ The following table lists the configurable parameters of the Moodle chart and th
+| `mariadb.auth.rootPassword`                 | Password for the MariaDB `root` user                                                                                  | _random 10 character alphanumeric string_                    |
+| `mariadb.auth.database`                     | Database name to create                                                                                               | `bitnami_moodle`                                             |
+| `mariadb.auth.username`                     | Database user to create                                                                                               | `bn_moodle`                                                  |
+| `mariadb.auth.password`                     | Password for the database                                                                                             | _random 10 character long alphanumeric string_               |
+| `mariadb.replication.enabled`               | MariaDB replication enabled                                                                                           | `false`                                                      |
+| `mariadb.primary.persistence.enabled`       | Enable database persistence using PVC                                                                                 | `true`                                                       |
+| `mariadb.primary.persistence.existingClaim` | Name of an existing `PersistentVolumeClaim` for MariaDB primary replicas                                              | `nil`                                                        |
+| `mariadb.primary.persistence.accessMode`    | Database Persistent Volume Access Modes                                                                               | `[ReadWriteOnce]`                                            |
+| `mariadb.primary.persistence.size`          | Database Persistent Volume Size                                                                                       | `8Gi`                                                        |
+| `mariadb.primary.persistence.storageClass`  | MariaDB primary persistent volume storage Class                                                                       | `nil` (uses alpha storage class annotation)                  |
+| `mariadb.primary.persistence.hostPath`      | Host mount path for MariaDB volume                                                                                    | `nil` (will not mount to a host path)                        |
+| `externalDatabase.user`                     | Existing username in the external db                                                                                  | `bn_moodle`                                                  |
```

Comment:
I think we have changed this value to `..username` in order to be aligned with `mariadb.auth.username`, up to you

Reply:
Changed to `.username`, thanks!